app/store/configureStore.js

```diff
@@ -12,19 +12,17 @@ import createLogger from 'redux-logger';
  *                          rendering.
  */
 export default function configureStore(initialState, history) {
-  const middleware = [thunk, promiseMiddleware];
   // Installs hooks that always keep react-router and redux
   // store in sync
-  const reactRouterReduxMiddleware = routerMiddleware(history);
+  const middleware = [thunk, promiseMiddleware, routerMiddleware(history)];
   if (__DEVCLIENT__) {
-    middleware.push(reactRouterReduxMiddleware, createLogger());
-  } else {
-    middleware.push(reactRouterReduxMiddleware);
+    middleware.push(createLogger());
   }
 
-  const finalCreateStore = applyMiddleware(...middleware)(createStore);
-
-  const store = finalCreateStore(rootReducer, initialState);
+  const store = createStore(rootReducer, initialState, compose(
+    applyMiddleware(...middleware),
+    typeof window === 'object' && typeof window.devToolsExtension !== 'undefined' ? window.devToolsExtension() : f => f
```

Comment:
I guess it depends on what you want. According to the repo, one of the advantages of it is:

> Having DevTools even in production without any drawbacks.

It's only problematic if the end user has redux devtools installed, and even then only if you care about them seeing the internals of your JS, which shouldn't really have any secrets in it anyway. I still kinda see the hesitation, but personally I think it's fine. Let me know what you think and I will change it accordingly